src/Framework/README.md

```diff
@@ -0,0 +1,25 @@
+# Framework
+
+Contains packages that provide reference assembles, default sets APIs for building ASP.NET Core applications, and assets used for self-contained deployments.
```

Comment:
I'm not sure what "default sets APIs" means. Please clarify. Please at least mention the shared framework and targeting pack installers that build on top of what's in this folder.

Further this doesn't make it clear the packages produced here exist _only_ to bundle assets created elsewhere and to add metadata.

Reply:
I'm not either, I copied these notes from the comments in the csproj for the assembly.  What's the correct summary?  Shared framework and targeting pack installers only?

Reply:
```suggestion
Contains packages that bundle reference and implementation assembles for Microsoft.AspNetCore.App.Runtime shared framework and for use in targeting pack installers.
```
Of course, this can probably be improved 😺